React/Views/RCTActivityIndicatorView.m

```diff
@@ -66,18 +66,23 @@ - (void)setActivityIndicatorViewStyle:(UIActivityIndicatorViewStyle)activityIndi
   }
 }
 
```

Comment:
Don't we still need a setting of the color property to result in kicking off a layer update? I think this is right, but we should double check that `setNeedsDisplay:` results in `updateLayer` being called on both iOS and macOS.
```suggestion
- (void)setColor: (UIColor*)color
{
  if (_color != color) {
    _color = color;
    [self setNeedsDisplay:YES];
  }
}
```

Reply:
Good idea. done.